store/details.go

```diff
@@ -41,6 +41,7 @@ type snapDetails struct {
 	Publisher        string             `json:"publisher,omitempty"`
 	RatingsAverage   float64            `json:"ratings_average,omitempty"`
 	Revision         int                `json:"revision"` // store revisions are ints starting at 1
+	ScreenshotURLs   []string           `json:"screenshot_urls,omitempty"`
```

Comment:
I've updated the tests in store_test.go to confirm that screenshots_urls gets converted correctly.

The main reason I wanted to add the width and height is so I can implement this support into snapd-glib now and not have to change API later. I put the width in height now so it wasn't confusing why we have a struct / use a JSON object for screenshots when they only contain one field (url). I figure also other stores might want to set this information potentially before the Snap store supports it.

The bug link at the top of this PR is the first step in the conversation with the store people.